server/daemonize_process.c

```diff
@@ -0,0 +1,98 @@
+#include "daemonize_process.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
```

Comment:
unistd.h may cause trouble on Windows. I was hoping for there to be a wrapper already, but can't find anything obvious at the moment.

I've used, for example, dlfcn-win32, which does a very good job with dlopen, dlsym, etc. 

With that in mind, I was hoping there would be more POISX wrappers for Windows. I'll keep digging.

There's nothing to change in this file now. Just that I will need to watch out for it when compiling on Windows.